allennlp_models/modelcards/rc-transformer-qa.json

```
@@ -38,7 +38,8 @@
         "model_performance_measures": "F1-score, Span Accuracy, Exact Match. Note that the metrics 
                                        that the model produces are calculated on a per-instance basis only.
                                        Since there could be more than one instance per question, these
-                                       metrics are not the official numbers on the SQuAD task.",
+                                       metrics are not the official numbers on the SQuAD task. To get
+                                       official numbers, run the script in scripts/transformer_qa_eval.py.",
```

I don't think this script exists anymore. Maybe just:

```suggestion
                                       official numbers, see https://rajpurkar.github.io/SQuAD-explorer/.",
```